racingcar-java/src/main/java/step3/domain/Car.java

```diff
@@ -0,0 +1,29 @@
+package step3.domain;
+
+import java.util.Random;
+
+public class Car {
+    private final static int RANDOM_MAX_BOUND = 10;
+    private final static int CAR_MOVE_POINT = 4;
+    private final static int DEFAULT_DISTANCE = 1;
+    private final static int INCREASE_DISTANCE_POINT = 1;
+    private final Random point = new Random();
+
+    private int number;
+    private int distance;
+
+    public Car(int number) {
+        this.number = number;
+        this.distance = DEFAULT_DISTANCE;
+    }
```

Comment:
```suggestion
    public Car(int number, int distance) {
        this.number = number;
        this.distance = distance;
    }
```
생성자는 필드의 모든 값을 주입 받을 수 있어야 한다고 생각이듭니다.    
그래야, 테스트 하는 과정에서 내부에서만 선언된 값에 대한 의존성이 줄어드니 말이죠..!!    

그렇다면 디폴트 값을 어디서 넣어줘야하는지 의문이 들텐데  
이 부분은 깊게 고민해보시길 바랍니다 :)

Reply:
이 부분이 저도 고민을 조금 했던 부분인데, 레이싱이라는것은 모두 동일한 조건에서 결승선을 향해 달려가는것이니 출발점은 기본적으로 모두 같아야한다는 생각에서 선언과 동시에 초기화를 해주는 방식을 채택했습니다.

알려주신대로 한다면 출발점이 달라질 가능성이 존재하지 않나요??

Reply:
오버로딩을 이용한다면 
현 구조 그대로 갈 수 있고

현재 테스트를 한다면 무조건 
초기값부터 테스트를 해야 할텐데
중간 상태에서 테스트를 하도록 바꿀수 있을 것 같아요